### ISSUE TITLE
Differences between the FAQ section on the new app and the FAQ section on the market-built app

### ISSUE THREAD

**Issue by hassan6190** [openedx/openedx-app-ios#233]
**Description:**
Profile screen > On the new app, the FAQ section shows a text (FAQ) whereas on the market build section, the FAQ section shows the text (View FAQ)

**Expected Behaviour:**
The FAQ section on both apps (New and Market build) should show the text "View FAQ"
![Improvement](https://github.com/openedx/openedx-app-ios/assets/69966132/bec310e5-9dd1-4339-a295-9f2a2125dabf)

**Comment by moiz994** [comment]
> @hassan6190 I'm not sure we need to add 'View' to 'FAQ' string because it has disclosure arrow now. @moiz994 could you check this from UX/design side please? Thank you

Can you please confirm if the new app is opening the FAQ page within the app or external browser? If it's leading to the external browser then it would be best to replicate the market build behaviour.

**Comment by rnr** [comment]
@moiz994 You are right - it opens in external. Ok let's change the string. Thank you.
